ci/dev/test.sh

```diff
@@ -3,12 +3,24 @@ set -euo pipefail
 
 main() {
   cd "$(dirname "$0")/../.."
-
-  cd test/test-plugin
+  pushd test/test-plugin
   make -s out/index.js
-  cd "$OLDPWD/test"
-  yarn
-  yarn test "$@"
+  # We must keep jest in a sub-directory. See ../../test/package.json for more
+  # information. We must also run it from the root otherwise coverage will not
+  # include our source files.
+  popd
+  if [[ -d ./test/node_modules ]]
+  then
+    echo "Found node_modules directory under /test"
+  else
+    echo "Did not find node_modules directory under /test"
+    echo "...Installing test dependencies."
+    pushd test
+    yarn
+    popd
+  fi
```

Comment:
Now that I'm thinking about it...Should we just modify the `postinstall` script to run `yarn` after installing the `lib/vscode` dependencies? 

Pro of that approach:
- no need to do this here

Pro of current approach:
- test.sh doesn't rely on a postinstall script

I could go either way